modules/http-impl/src/main/java/org/mule/services/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -255,11 +249,43 @@ private int retrieveMaximumHeaderSectionSize() {
     try {
       return valueOf(getProperty(MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY, String.valueOf(DEFAULT_MAX_HTTP_PACKET_HEADER_SIZE)));
     } catch (NumberFormatException e) {
-      throw new MuleRuntimeException(CoreMessages.createStaticMessage(String.format("Invalid value %s for %s configuration",
-                                                                                    getProperty(MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY),
-                                                                                    MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY)),
+      throw new MuleRuntimeException(createStaticMessage(String.format("Invalid value %s for %s configuration",
+                                                                       getProperty(MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY),
+                                                                       MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY)),
                                      e);
     }
   }
 
+  private int convertToSeconds(int milliseconds) {
+    return (int) Math.ceil((double) milliseconds / 1000.0);
+
+  }
+
+  private class ProxyGrizzlyHttpServer extends GrizzlyHttpServer {
```

Comment:
No, maybe it should have a better name. It's just a wrapper to handle the ServerManager specific disposal of per server objects.